### PR TITLE
fix some shift clicks not calling GUIClickEvent

### PR DESCRIPTION
**Pull request by l4zs**

### CODE REVIEW

**Comment by l4zs** [comment]
was a bit confused about PICKUP_SOME and PICKUP_ONE apparently those are called if you use oversized stacks so should be added too I guess

**Comment by NyCodeGHG** [comment]
Is this intended?

File: src/main/kotlin/net/axay/kspigot/gui/GUIHolder.kt [hunk 1]
```diff
@@ -77,7 +77,7 @@ object GUIHolder : AutoCloseable {
 }
 
 private val InventoryAction.isGUIClick
-    get() = this == InventoryAction.PICKUP_ALL || this == InventoryAction.PICKUP_HALF
+    get() = this == InventoryAction.PICKUP_ALL || this == InventoryAction.PICKUP_HALF || this == InventoryAction.PICKUP_SOME || this == InventoryAction.PICKUP_ONE || this == InventoryAction.MOVE_TO_OTHER_INVENTORY
```
Comment (bluefireoly) [inline src/main/kotlin/net/axay/kspigot/gui/GUIHolder.kt:80]: Could you also add a new line after this extension value? This broke due to my reformat commit.

**Review by bluefireoly — APPROVED** [review]

**Comment by bluefireoly** [comment]
> Is this intended?

I actuall don't know why I did not add these action types when I wrote this, if it had a particular reason I guess we will find it out sooner or later haha

**Comment by bluefireoly** [comment]
Thank you!

**Comment by NyCodeGHG** [comment]
> > Is this intended?
> 
> I actually don't know why I did not add these action types when I wrote this, if it had a particular reason I guess we will find it out sooner or later haha

I actually meant the 4 commits with the same message. @l4zs and I wrote our messages at the same time.

**Pull request merged**